iidm/iidm-api/src/main/java/com/powsybl/iidm/network/TapChangerAdder.java

```diff
@@ -10,7 +10,7 @@
 /**
  * @author Florent MILLOT {@literal <florent.millot at rte-france.com>}
  */
-public interface TapChangerAdder<S extends TapChangerAdder<S, A, C>, A extends TapChangerStepAdder<A, S>, C extends TapChanger<C, ?>> {
+public interface TapChangerAdder<S extends TapChangerAdder<S, A, C>, A extends TapChangerStepAdder<A, S>, C extends TapChanger<C, ?, ?>> {
```

Comment:
And we can try to remove those ones also ?

Reply:
Done